lua/cmake-tools/init.lua

```diff
@@ -83,7 +83,7 @@ function cmake.generate(opt, callback)
   -- if exists presets, preset include all info that cmake
   -- needed to execute, so we don't use cmake-kits.json and
   -- cmake-variants.[json|yaml] event they exist.
-  local presets_file = presets.check(config.cwd)
+  local presets_file = config.base_settings.use_preset and presets.check(config.cwd)
```

Comment:
It should be `use_presets`?

Reply:
I was trying to follow the other variable names like `configure_preset`, `build_preset`, `select_configure_preset`, `configure_preset_names`, `format_preset_name`, etc.

And since you cannot use more than 1 preset at a time, I also thought that `use_preset` would be ok.

In all honesty, I have no preference for the singular or plural version of it. Do you want me to change everything for the "plural" version?

Reply:
I think `use_preset` is better. But the problem is you mixed the use with `use_presets` and `use_preset`.

Reply:
oh, I did not realize that 🤦🏼 

Will fix in a bit

Reply:
Done, it should all be `use_preset`.